envs/arrow-env.yaml

```diff
@@ -32,3 +32,5 @@ packages:
 {% endif %}
   - feedstock : grpc-cpp
   - feedstock : pyarrow
+
+git_tag_for_env: open-ce-v1.4.1
```

Comment:
shouldn't this be `v1.5.1` ?

Reply:
yes @alexlang74. I will remove this and It's not added in our 1.5.1 build update.

Reply:
Thanks, @ritumd !